app/main/posts/posts-routes.js

```diff
@@ -235,13 +260,14 @@ function (
             template: require('./modify/main.html')
         }
     )
-    .state(
-        {
-            name: 'postEdit',
-            url: '/posts/:id/edit',
-            controller: require('./modify/post-edit.controller.js'),
-            template: require('./modify/main.html')
-        }
-    );
+    // .state(
```

Comment:
can we just remove this ? or do we want it here for some particular reason?

Reply:
Didn't have time to check if it was being used yet